libr/core/cmd_anal.c

```diff
@@ -971,26 +973,55 @@ static void var_help(RCore *core, char ch) {
 	}
 }
 
-static void var_accesses_list(RAnalFunction *fcn, RAnalVar *var, int access_type) {
+static void var_accesses_list(RAnalFunction *fcn, RAnalVar *var, int access_type, const char *name) {
 	RAnalVarAccess *acc;
 	bool first = true;
 	r_vector_foreach (&var->accesses, acc) {
 		if (!(acc->type & access_type)) {
 			continue;
 		}
-		if (!first) {
+		if (first) {
```

Comment:
why here? this should probably be print always, as first thing, so it can be printed before the `r_vector_foreach` instead of here.

Reply:
because i only want to print varnames that have xrefs, otherwise it prints an empty line with trailing spaces

Reply:
but this changes the behaviour of commands like `afvR` and `afvR`.
I think it is useful to have things like:
```
[0x00004e60]> afvR
      argc  0x4e6d
      argv  0x4e70,0x4e77
   var_40h  0x5d3c
   var_58h  0x5031,0x5096,0x6113,0x6614,0x664e
   var_60h  0x6063,0x610b,0x65ec,0x6603,0x663b
   var_50h  
   var_43h  0x60cc
   var_41h  
   var_42h  
   var_56h  0x504f
```

Because it let's you see var_50h is not accessed for reading.With your changes, AFAIU, you won't see the `var_50h` line and I think it hides some useful info.